content/neuvector-prime/azure/contents.lr

```diff
@@ -0,0 +1,464 @@
+    configuration settings in Azure Portal. Until this is resolved, it is suggested
+    to change the Admin password after initial login.  Edit profile in the NeuVector
+    console to change password.
+
+    ---
+
+
+
+1. Access NeuVector Console as an admin user.
+2. Click the "Admin user" drop-down and click "My Profile".
+3. Click "Edit Profile" and set a new password and click "Update Profile".
+
+
+
+
+
+
+   !["Change Admin Password"](./changepwd.png)
+
+
+
+
+
+
+
+
+**IMPORTANT!**
+
+    ---
+    Do not change or save the cluster configuration settings for Fullname, Password,
+    or Role by changing the key value pairs :
+
+           "controller.secret.data.userinitcfg\\.yaml.users[0].Fullname"
+           "controller.secret.data.userinitcfg\\.yaml.users[0].Password"
+           "controller.secret.data.userinitcfg\\.yaml.users[0].Role"
+
+    Changing this value does not update the NeuVector Admin Fullname, Password,
+    or Role. This will cause unnecessary cluster restarts.
+
+    ---
+
+
+
+
+
+
+
+   !["Keyvaluepair config"](./keyvaluepairconfig.png)
+
+## NeuVector billing
+
+The billing will be available in "Azure Portal billing".
+
+Go to Home > Cost Management <subscription> | Cost analysis.
+
+
+## Uninstalling NeuVector and scanner updater
+
+Before uninstalling NeuVector, uninstall the scanner updater.
+Execute the following command in Cloud Shell or Node which has access to kubectl.
+
+```
+kubectl delete -f scanner.yaml
+```
+The output should be similar to the example below:
+```
+deployment.apps "neuvector-scanner-pod" deleted
+cronjob.batch "neuvector-updater-pod" deleted
+```
+
+Check the scanner pods are running:
+```
+kubectl get pods -n neuvector
+```
+There should be no neuvector-scanner-pod listed in the output.
+
+```
+NAME                                        READY   STATUS    RESTARTS   AGE
+neuvector-controller-pod-754c55f668-5vnxv   1/1     Running   0          3h7m
+neuvector-controller-pod-754c55f668-658vb   1/1     Running   0          3h8m
+neuvector-controller-pod-754c55f668-bfk2c   1/1     Running   0          3h10m
+neuvector-csp-pod-cb675b47c-wnm8g           1/1     Running   0          3h15m
+neuvector-enforcer-pod-2d9qd                1/1     Running   0          3h15m
+neuvector-enforcer-pod-5lgw4                1/1     Running   0          3h15m
+neuvector-enforcer-pod-8qxdb                1/1     Running   0          3h15m
+neuvector-enforcer-pod-9gngc                1/1     Running   0          3h15m
+neuvector-enforcer-pod-9jnt7                1/1     Running   0          3h15m
+neuvector-manager-pod-59677c8ccb-4g9hj      1/1     Running   0          3h10m
+```
+
+To navigate to and uninstall NeuVector in the Azure Portal:
+
+
+1. Click Home > Kubernetes Services
+2. Click the Cluster name
+3. Click Settings > Extension + application
+4. Select NeuVector Extension and click Uninstall
+
+
+
+!["Uninstall extension"](./uninstall.png)
+
+
```

Comment:
maybe also add the command to uninstall  `az k8s-extension delete` and the helm steps

```
helm  ls -n neuvector 
helm uninstall -n neuvector <NAME from the helm ls command and what is show in the UI>

 in case the UI uninstall fails. or put this in the troubleshooting section.

Reply:
we should tell them to do this first

`az k8s-extension delete --name <extension name>   --cluster-name <clustername>  --resource-group <resource group> --cluster-type managedClusters -y`

they can run `helm ls -n neuvector <extension name>`  to verify it was removed and if not they can force remove it with.

`helm uninstall -n neuvector  <extension name>`

They get the extension name from the UI or this command  This screen shot  shows the name (truncated) [content/neuvector-prime/azure/enabled.png](https://github.com/SUSE-Enceladus/marketplace-docs/pull/32/files#diff-1130b9c126b0b4bbafb59c4de48053f43d487281757fee9edc75e270e6b12ce9)

`az k8s-extension list   -g <resource group> --cluster-type managedClusters -c <clustername>`

Reply:
I think we should use this command firsrt

`az k8s-extension delete --name <extension name> --cluster-name <clustername> --resource-group <resource group> --cluster-type managedClusters -y` 

before doing the helm. helm should be the last resort